setup.py

```diff
@@ -8,27 +8,16 @@
 
 from setuptools import setup
 from pathlib import Path
-import subprocess
-import re
+
+STD_PACKAGES = set(('array', 'math', 'os', 'random', 'struct', 'sys', 'ssl', 'time'))
 
 STD_PACKAGES = set(('array', 'math', 'os', 'random', 'struct', 'sys', 'ssl', 'time'))
```

Comment:
this looks funky

Reply:
Ah yep  I think  this got doubled up, it's likely I misclicked it while working through changes to merge from main. I'll make a new PR to remove the dupe